components/usage/pkg/stripe/stripe.go

```diff
@@ -69,11 +69,7 @@ type Invoice struct {
 // UpdateUsage updates teams' Stripe subscriptions with usage data
 // `usageForTeam` is a map from team name to total workspace seconds used within a billing period.
 func (c *Client) UpdateUsage(ctx context.Context, creditsPerAttributionID map[db.AttributionID]int64) error {
-	attributionIDs := make([]db.AttributionID, 0, len(creditsPerAttributionID))
-	for k := range creditsPerAttributionID {
-		attributionIDs = append(attributionIDs, k)
-	}
-	queries := queriesForCustomersWithAttributionIDs(attributionIDs)
```

Comment:
Shifted this into `queriesForCustomersWithAttributionIDs` as it was making the main function a bit harder to read.